README.md

```diff
@@ -4,8 +4,18 @@ This repository contains tooling for reviewing Mattermost databases with complex
 
 ### Building
 
-To build and install, run the following:
+Before building the `mmdbt` CLI, ensure that you have the latest version of Go 1.22 installed.
```

Comment:
Should we link to the version in go.mod instead of explicitly stating 1.22, so that it's not missed in future version changes?

Reply:
Oh yeah, I like that idea. Let me change it.